tools/build_file_generator/third_party_deps/ThirdPartyDepsAnalyzer.java

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
Did you change this code, or just move it?

Reply:
The code has lots of changes. I created draft PR to see all them - https://github.com/google/startup-os/pull/557/files